### PR TITLE
Using get from model method in PCA

### PR DESCRIPTION
**Pull request by stevekogo**

### CODE REVIEW

File: instat/sdgPrincipalComponentAnalysis.vb [hunk 1]
```diff
@@ -27,24 +27,31 @@ Public Class sdgPrincipalComponentAnalysis
...
+        clsRScores.SetRCommand(frmMain.clsRLink.strInstatDataObject & "$get_from_model")
+        clsRScores.AddParameter("model_name", Chr(34) & dlgPrincipalComponentAnalysis.modelname & Chr(34))
+        clsRScores.AddParameter("value1", Chr(34) & "ind" & Chr(34))
+        clsRScores.AddParameter("value2", Chr(34) & "coord" & Chr(34))
+        frmMain.clsRLink.RunScript(clsRScores.ToScript(), 2)
     End Sub
 
     Private Sub Residuals()
-        'frmMain.clsRLink.RunScript(dlgPrincipalComponentAnalysis.ucrBasePCA.clsRsyntax.GetScript() & "$eig", 2)
+        'TODO
```
Comment (dannyparsons) [inline instat/sdgPrincipalComponentAnalysis.vb]: Can this also be done in the same way?

File: instat/dlgPrincipalComponentAnalysis.vb [hunk 2]
```diff
@@ -17,6 +17,8 @@ Imports instat.Translations
 Public Class dlgPrincipalComponentAnalysis
     Public bFirstLoad As Boolean = True
     Public ExplanatoryVariables
+    Public modelname As String = ""
```
Comment (dannyparsons) [inline instat/dlgPrincipalComponentAnalysis.vb]: This should be in the correct format: strModelName

File: instat/sdgPrincipalComponentAnalysis.vb (hunk 1 above)
Comment (stevekogo) [inline instat/sdgPrincipalComponentAnalysis.vb]: I need to discuss with @rdstern before proceeding with this

**Comment by stevekogo** [comment]
@dannyparsons I have implemented the changes

File: instat/sdgPrincipalComponentAnalysis.vb (hunk 1 above)
Comment (dannyparsons) [inline instat/sdgPrincipalComponentAnalysis.vb]: Ok, he'll be back on Monday, you could discuss it with him then.

**Pull request merged**